Tests/UnitTests/Core/Vertexing/AdaptiveMultiVertexFitterTests.cpp

```diff
@@ -213,20 +213,18 @@ BOOST_AUTO_TEST_CASE(adaptive_multi_vertex_fitter_test) {
     int vtxIdx = (int)(iTrack / nTracksPerVtx);
     state.vtxInfoMap[&(vtxList[vtxIdx])].trackLinks.push_back(
         &(allTracks[iTrack]));
-    state.tracksAtVerticesMap.insert(
-        std::make_pair(std::make_pair(&(allTracks[iTrack]), &(vtxList[vtxIdx])),
-                       TrackAtVertex<BoundTrackParameters>(
-                           1., allTracks[iTrack], &(allTracks[iTrack]))));
+    state.tracksAtVerticesMap.insert(std::make_pair(
+        std::make_pair(&(allTracks[iTrack]), &(vtxList[vtxIdx])),
+        TrackAtVertex<BoundTrackParameters>(&(allTracks[iTrack]), 1.)));
```

Comment:
```suggestion
        std::make_pair(&allTracks[iTrack], &vtxList[vtxIdx]),
        TrackAtVertex<BoundTrackParameters>(&allTracks[iTrack], 1.)));
```
nitpick: that `()` syntax looks a bit weird to me. same below